src/guide/list.md

```diff
@@ -268,25 +268,23 @@ Similar to template `v-if`, you can also use a `<template>` tag with `v-for` to
 Note that it's **not** recommended to use `v-if` and `v-for` together. Refer to [style guide](../style-guide/#avoid-v-if-with-v-for-essential) for details.
 :::
 
-When they exist on the same node, `v-for` has a higher priority than `v-if`. That means the `v-if` will be run on each iteration of the loop separately. This can be useful when you want to render nodes for only _some_ items, like below:
+When they exist on the same node, `v-if` has a higher priority than `v-for`. That means that if `v-if` condition depends on the property from `v-for` scope, we will have an error: 
 
 ```html
 <li v-for="todo in todos" v-if="!todo.isComplete">
+<!-- Property "todo" was accessed during render but is not defined on instance. -->
   {{ todo }}
 </li>
 ```
 
-The above only renders the todos that are not complete.
-
-If instead, your intent is to conditionally skip execution of the loop, you can place the `v-if` on a wrapper element (or [`<template>`](conditional#conditional-groups-with-v-if-on-lt-template-gt)). For example:
+This can be fixed with moving `v-for` to the wrapping <template> tag:
```

Comment:
The `<template>` needs backticks. I'd also make a few grammatical tweaks:

```text
This can be fixed by moving the `v-for` to a wrapping `<template>` tag:
                  ^^        ^^^            ^          ^^^^^^^^^^^^
```

Reply:
I don't think we should remove this line altogether because we need to show that this code snippet is incorrect. I changed a message to reflect this better

Reply:
Thank you, fixed 👍🏻